pyproject.toml

```
@@ -29,8 +29,8 @@ dependencies = [
     "pydantic~=2.0,>=2.10.1,<2.10.2",
     "tiktoken>=0.4.0",
     "typing-extensions; python_version <= '3.11'",  # for typing.override
-    'litellm; python_version < "3.13"',  # NOTE: paper-qa==5.3 doesn't support 3.13 yet
     'litellm>=1.49.1; python_version >= "3.13"',  # For removal of imghdr
+    'litellm>v1.59.3; python_version < "3.13"',  # NOTE: paper-qa==5.3 doesn't support 3.13 yet
```

Okay, so I think actually due to the way you use `hasattr`, we actually don't need to pin this in the required dependencies

We can pin latest `litellm` in dev dependencies here, but we don't need to pin in the core deps

Does that make sense?

---

It kind of does. But if a user doesn't install the `dev` dependencies, `reasoning_content` will never be populated (due to the `litellm` version), and it'll be like a silent error. 
I'm down to go with that -- we just need to alert people they need to use the dev extras